### PR TITLE
OHRM5X-138: General Info isuues

### PR DESCRIPTION
**Pull request by KaveeshaSilva1996**

### CODE REVIEW

File: symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss [hunk 1]
```diff
@@ -1,3 +1,27 @@
-.organization-name-container{
+.organization-name-container {
   grid-column: span 2;
 }
+
+.no-of-employees-value {
+  font-family: "Nunito Sans";
```
Comment (samanthajayasinghe) [inline symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss:6]: can't use genric scss vars here?

File: symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+  font-weight: 400;
+  line-height: 1.5;
+  border-radius: 0.65rem;
+  padding: 0.675rem 0.5rem;
+  width: 100%;
+  min-height: 45px;
+}
+
+.no-of-employees-label {
+  font-family: "Nunito Sans";
```
Comment (samanthajayasinghe) [inline symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss:23]: here too

File: symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue [hunk 3]
```diff
@@ -44,11 +44,14 @@
               />
             </oxd-grid-item>
             <oxd-grid-item>
-              <oxd-input-field
-                label="Number of Employees"
-                v-model="organization.noOfEmployees"
-                :disabled="true"
-              />
+              <div>
+                <oxd-label class="no-of-employees-label">
```
Comment (Super-Chama) [inline symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue:48]: @KaveeshaSilva1996 No need to use custom elements. we can utilise OXD components.

```
<oxd-grid-item>
  <oxd-input-group label="Number of Employees">
    <oxd-text tag="h5">
      {{ organization.noOfEmployees }}
    </oxd-text>
  </oxd-input-group>
</oxd-grid-item>
```

File: symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss (hunk 1 above)
Comment (Super-Chama) [reply symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/general-info.scss:6]: avoid custom css as much as possible. with use of oxd components, you won't need these styles.

File: symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue [hunk 4]
```diff
@@ -44,11 +44,11 @@
               />
             </oxd-grid-item>
             <oxd-grid-item>
-              <oxd-input-field
-                label="Number of Employees"
-                v-model="organization.noOfEmployees"
-                :disabled="true"
-              />
+              <oxd-input-group label="Number of Employees">
+                <oxd-text tag="h5" :style="{ fontSize : '12px', padding: '0.675rem 0.5rem' }">
```
Comment (RajithaKumara) [inline symfony/client/src/orangehrmAdminPlugin/pages/organizationGeneralInformation/ViewOrganizationGeneralInformation.vue:48]: No inline styles, please move to css class

File: symfony/plugins/orangehrmPimPlugin/Dao/EmployeeDao.php [hunk 5]
```diff
@@ -228,11 +228,17 @@ public function getSubordinateIdListBySupervisorId(
...
 
     /**
+     * @param bool $includeTerminated
      * @return int
      */
-    public function getNumberOfEmployees(): int
+    public function getNumberOfEmployees(bool $includeTerminated = false): int
     {
         $q = $this->createQueryBuilder(Employee::class, 'e');
+
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmPimPlugin/Dao/EmployeeDao.php:237]: How about the purged employees

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: symfony/plugins/orangehrmPimPlugin/Dao/EmployeeDao.php (hunk 5 above)
Comment (KaveeshaSilva1996) [reply symfony/plugins/orangehrmPimPlugin/Dao/EmployeeDao.php:237]: Not implemented in 4.x

**Pull request merged**